CONTRIBUTING.md

```diff
@@ -67,10 +67,10 @@ To build, run and test locally, this checklist below can help solve most issues.
 - Verify Node version. This client should match the Node version requirement from `itwinjs-core` as described here in `nodeSupportedVersionRange` : <https://github.com/iTwin/itwinjs-core/blob/master/rush.json>
 
 Example commands below to install, clean, build and test :
-- `npm i`
+- `pnpm install`
 - `npm run clean`
 - `npm run build`
 - `npm run lint`
-- `npm run test:integration` to run integration tests locally.
+- `npm run integration-tests`
```

Comment:
for reality-data-client package.json the npm command might still be `npm run test:integration` , you can change or add it. this might impact github actions as well